From f2680b3cd14bd5446cc8eba72357583bbe96b814 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Wed, 16 Feb 2022 14:22:25 -0700
Subject: [PATCH] Add site.cfg

---
 site.cfg | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 site.cfg

diff --git a/site.cfg b/site.cfg
new file mode 100644
index 0000000..d76db8d
--- /dev/null
+++ b/site.cfg
@@ -0,0 +1,2 @@
+[phono3py]
+extra_compile_args = -fopenmp
